packages/blockchain-wallet-v4-frontend/src/scenes/Airdrops/StxAirdrop/index.tsx

```diff
@@ -49,77 +53,130 @@ export type Props = {
   tags: TagsType
 }
 
+const CompletedStxInfo = () => (
+  <CompletedWrapper>
+    <Header style={{ marginBottom: '16px' }}>
+      <Text size='20px' color='white' weight={600}>
+        <FormattedMessage
+          id='scenes.airdrops.blockstack.wallet.title'
+          defaultMessage='My Blockstack Wallet'
+        />
+      </Text>
+    </Header>
+    <Text size='14px' color='white' weight={700}>
+      <FormattedMessage
+        id='scenes.airdrops.blockstack.wallet.q1'
+        defaultMessage='Where are my Stacks?'
+      />
+    </Text>
+    <TextParagraph size='13px' color='white' weight={500}>
+      <FormattedMessage
+        id='scenes.airdrops.blockstack.wallet.a1'
+        defaultMessage='Your Stacks (STX) are saved in your Blockchain Wallet.'
+      />
+    </TextParagraph>
+    <TextParagraph size='13px' color='white' weight={500}>
+      <FormattedMessage
+        id='scenes.airdrops.blockstack.wallet.a2'
+        defaultMessage='Please note the balance is currently non-transferable.'
+      />
+    </TextParagraph>
+    <Text size='13px' color='white' weight={500}>
+      <FormattedMessage
+        id='scenes.airdrops.blockstack.wallet.a3'
+        defaultMessage='Learn more about this and future wallet support for STX'
+      />
+      {' '}
+      <Link
+        href='https://support.blockchain.com/hc/en-us/articles/360038745191'
+        target='_blank'
+        size='13px'
+        color='white'
+        weight={500}
+        style={{ textDecoration: 'underline' }}
+      >
+        <FormattedMessage
+          id='scenes.airdrops.blockstack.wallet.here'
+          defaultMessage='here'
+        />
+      </Link>
+      {'.'}
+    </Text>
+  </CompletedWrapper>
+)
+
 const StxAirdrop = (props: Props & LinkDispatchPropsType) => {
-  return (
-    <Box>
-      <div>
-        <Header>
-          <Icon name='stx' color='stx' size='32px' />
+  const blockstackCampaign = props.userCampaignsInfoResponseList.find(
+    (campaign: CampaignInfoType) => campaign.campaignName === 'BLOCKSTACK'
+  )
+  return propEq('userCampaignState','NONE', blockstackCampaign) ? null : (
+    propEq('userCampaignState','REWARD_RECEIVED', blockstackCampaign) ? (
+      <CompletedStxInfo />
+    ) : (
```

Comment:
UserCampaignState =
  | 'FAILED'
  | 'REWARD_RECEIVED'
  | 'TASK_FINISHED'
  | 'REWARD_SEND'
  | 'REGISTERED'
  | 'NONE'

This means:
if userCampaignState is NONE show nothing
if userCampaignState is REWARD_RECEIVED show the new thing
if userCampaignState is FAILED, TASK_FINISHED, REGISTERED, or REWARD_SEND we show the old Box with StxStatus and that has calls to action. Are you sure we want that?

Reply:
hmm... yeah not sure about the last one